### PR TITLE
ENG-11827: Expand user dir

### PR DESCRIPTION
**Pull request by akhanzode**

### CODE REVIEW

File: lib/python/voltcli/voltdb.d/start.py [hunk 1]
```diff
@@ -43,7 +45,8 @@
 )
 def start(runner):
     if runner.opts.directory_spec:
-        runner.args.extend(['voltdbroot', runner.opts.directory_spec])
+        upath = os.path.expanduser(runner.opts.directory_spec)
```
Comment (xinjiacs) [inline lib/python/voltcli/voltdb.d/start.py]: @akhanzode 
Do you care about any input space, like "   ~/my_dir"? 
Will this work?

Comment (akhanzode) [inline lib/python/voltcli/voltdb.d/start.py]: this is exactly to address ~/dir situation are you asking if someone gives it in ""?

Comment (xinjiacs) [inline lib/python/voltcli/voltdb.d/start.py]: "~/mydir" is a little different from "     ~/mydir"? Not sure if that's possible. I know "os.path.expanduser" does not work with any dir leading without "~".

Comment (akhanzode) [inline lib/python/voltcli/voltdb.d/start.py]: That would not be a userdir expandable dir.

**Pull request merged**